<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AudioTailTrimmer</class>
 <widget class="QDialog" name="AudioTailTrimmer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>698</width>
    <height>609</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>尾部冗余静音裁剪工具</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btnImportFiles">
       <property name="text">
        <string>导入音频文件</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnAnalyzeTails">
       <property name="text">
        <string>分析冗余时长</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnStartTrim">
       <property name="text">
        <string>开始裁剪音尾</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblCutThreshold">
       <property name="text">
        <string>静音判断阈值</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spbCutThreshold">
       <property name="suffix">
        <string>dB</string>
       </property>
       <property name="minimum">
        <number>-96</number>
       </property>
       <property name="maximum">
        <number>0</number>
       </property>
       <property name="value">
        <number>-48</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblFadeDuration">
       <property name="text">
        <string>淡出时长</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="spbFadeDuration">
       <property name="suffix">
        <string>s</string>
       </property>
       <property name="minimum">
        <double>0.010000000000000</double>
       </property>
       <property name="maximum">
        <double>1.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.010000000000000</double>
       </property>
       <property name="value">
        <double>0.200000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="tblFileList">
     <property name="dragEnabled">
      <bool>true</bool>
     </property>
     <property name="dragDropMode">
      <enum>QAbstractItemView::DragDrop</enum>
     </property>
     <column>
      <property name="text">
       <string>音频文件名</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>总时长</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>尾部静音时长</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>处理状态</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
